components/list/list-item-drag-drop-mixin.js

```diff
@@ -158,8 +158,10 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 	}
 
 	_onDragStart(e) {
-		e.dataTransfer.setData('text/plain', `${this.dropText}`);
 		e.dataTransfer.effectAllowed = 'move';
+		if (e.dataTransfer.setData && this.dropText) {
+			e.dataTransfer.setData('text/plain', `${this.dropText}`);
```

Comment:
what if condition is evaluated to false, how it will affect user experience and drag/drop functionality?

Reply:
It's an extra neat thing I put in. At the end of the day it can be removed.